modules/installation-gcp-config-yaml.adoc

```diff
@@ -37,21 +37,15 @@ controlPlane: <2>
   name: master
   platform:
     gcp:
-      rootVolume:
-        iops: 4000
-        size: 500
-        type: io1
+      project: project-name
```

Comment:
Per https://github.com/openshift/installer/blob/master/docs/user/gcp/customization.md#machine-pools, machine pools for GCP support **type** and **zones** only without **project**. The available zones for a specific region can be found https://cloud.google.com/compute/docs/regions-zones